willa/chatbot/graph_manager.py

```diff
@@ -11,13 +11,9 @@
 from langgraph.graph.state import CompiledStateGraph
 from langgraph.graph.message import AnyMessage
 from langmem.short_term import SummarizationNode # type: ignore
-
 from willa.config import CONFIG, get_lance, get_model
```

Comment:
We should be able to use one import line for all the config needs.

```suggestion

from willa.config import CONFIG, get_initial_prompt, get_lance, get_model
```

Reply:
Thanks, good catch